integration-tests/tests/scrape-prom-metrics/scrape_prom_metrics_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build !windows
```

Comment:
The problem with the integration tests is that they cannot run in Drone. They only run in github Actions (because it uses docker compose which is not possible in Drone). I was planning to use test-containers (see https://github.com/grafana/agent/pull/5861) but now that we are moving away from Drone I will probably keep them in Github Actions. The reason why the tests are running is because the Windows pipeline is calling "go test" at the root of the pipeline and that triggers the integration tests (https://github.com/grafana/agent/blob/main/.drone/drone.yml#L184). They dont run when you run make test because I made a special exclusion (https://github.com/grafana/agent/blob/main/Makefile#L176). I wanted to do the same thing to fix the windows pipeline but I modified directly the yml file not knowing that it was a generated file (I just realised now) https://github.com/grafana/agent/pull/5643/files. Now Im not sure whether this fix actually work or if that's the right approach (@tpaschalis suggested also to use build tags but you need to make sure that the nothing runs in the integration tests folder)